app/code/Magento/MediaGallery/Model/Asset/Command/Save.php

```diff
@@ -72,37 +64,24 @@ public function execute(AssetInterface $mediaAsset): int
 
             $connection->insertOnDuplicate(
                 $tableName,
-                $this->filterData($this->objectProcessor->buildOutputDataArray($mediaAsset, AssetInterface::class))
+                [
+                    'id' => $mediaAsset->getId(),
+                    'path' => $mediaAsset->getPath(),
+                    'title' => $mediaAsset->getTitle(),
+                    'source' => $mediaAsset->getSource(),
+                    'content_type' => $mediaAsset->getContentType(),
+                    'width' => $mediaAsset->getWidth(),
+                    'height' => $mediaAsset->getHeight(),
+                    'size' => $mediaAsset->getSize(),
+                    'created_at' => $mediaAsset->getCreatedAt(),
+                    'updated_at' => $mediaAsset->getUpdatedAt(),
```

Comment:
Update at and created at should not be saved to the database if their values are `null`